src/actions/resetPasswordAction.jsx

```diff
@@ -0,0 +1,48 @@
+import swal from "sweetalert2";
+
+export const sendEmailLink = data => dispatch => {
+  /*Sends a pasword reset link*/
+  return fetch("http://localhost:8000/api/users/password_reset/", {
```

Comment:
Can you kindly remove all the localhost urls from your actions .

Reply:
@chucky, fixing all reviews